src/main/scala/io/sdkman/changelogs/MavenMigration.scala

```diff
@@ -5,15 +5,25 @@ import com.mongodb.client.MongoDatabase
 
 @ChangeLog(order = "013")
 class MavenMigration {
+  @ChangeSet(order = "020", id = "020-add_maven_3.8.6", author = "aalmiray")
+  def migration020(implicit db: MongoDatabase): Unit = {
+    Version(
+      "maven",
+      "3.8.6",
+      "https://archive.apache.org/dist/maven/maven-3/3.8.6/binaries/apache-maven-3.8.6-bin.zip"
+    ).validate()
+      .insert()
+      .asCandidateDefault()
+  }
+  
   @ChangeSet(order = "009", id = "009-add_maven_3.8.5", author = "aalmiray")
-  def migration008(implicit db: MongoDatabase): Unit = {
+  def migration009(implicit db: MongoDatabase): Unit = {
```

Comment:
We can probably remove all the old unused migrations.

Reply:
Leaving only the last one for `3.8.6`? I'll do it in a jiffy.